policy/modules/services/postfix.te

```diff
@@ -13,6 +13,15 @@ policy_module(postfix, 1.23.2)
 ## </desc>
 gen_tunable(postfix_local_write_mail_spool, true)
 
+## <desc>
+##	<p>
+##	Determine whether postfix can
+##	communicate with dovecot for
+##	authentication and delivery.
+##	</p>
+## </desc>
+gen_tunable(postfix_use_dovecot, false)
```

Comment:
I would personally probably make this access unconditional, but that is subjective

Reply:
I agree that I can't personally think of any situation where one may want to have postfix and dovecot installed simultaneously yet not configured to interoperate, I think there's still some value in having this behavior conditional. What about enabling it by default instead?

Reply:
I agree with Dominick on this. The access isn't really dangerous.

Reply:
OK.